judge/models/comment.py

```diff
@@ -166,7 +166,7 @@ def get_absolute_url(self):
         return '%s#comment-%d' % (self.link, self.id)
 
     def __str__(self):
-        return '%(page)s by %(user)s' % {'page': self.page, 'user': self.author.user.username}
+        return _('%(comment_id)s by %(user)s') % {'comment_id': self.page, 'user': self.author.user.username}
```

Comment:
Wrong. `self.page` is the page the comment is posted to, not the ID of the comment.

Reply:
Fixed.